### PR TITLE
Add websocket ping variables

### LINKED ISSUE #1904: Ability to set ping_interval and ping_timeout parameters for WebSocketCommonProtocol
**Is your feature request related to a problem? Please describe.**
The `ping_interval` and `ping_timeout` parameters for [WebSocketCommonProtocol](https://websockets.readthedocs.io/en/stable/api.html#websockets.protocol.WebSocketCommonProtocol) default to 20 seconds. It would be nice to be able to set them with configuration values.

**Describe the solution you'd like**
Add `WEBSOCKET_PING_INTERVAL` and `WEBSOCKET_PING_TIMEOUT` to configuration values and pass the values into the [WebSocketCommonProtocol initialization](https://github.com/huge-success/sanic/blob/master/sanic/websocket.py#L116).

**Additional context**

### PR DESCRIPTION
**Pull request by shawnhill**
Resolves #1904.
Adds `WEBSOCKET_PING_TIMEOUT` and `WEBSOCKET_PING_INTERVAL` config values. This is my first contribution to Sanic.  I couldn't find where the existing websocket config values were being set on the WebSocketProtocol. Please advise if I missed something. Added a test to ensure all config values present when WebSocketProtocol created.

### CODE REVIEW

File: docs/sanic/websocket.rst [hunk 1]
```diff
@@ -51,5 +51,6 @@ You could setup your own WebSocket configuration through ``app.config``, like
     app.config.WEBSOCKET_MAX_QUEUE = 32
     app.config.WEBSOCKET_READ_LIMIT = 2 ** 16
     app.config.WEBSOCKET_WRITE_LIMIT = 2 ** 16
+    app.config.WEBSOCKET_PING_INTERVAL = 20
```
Comment (ahopkins) [inline docs/sanic/websocket.rst:55]: `WEBSOCKET_PING_TIMEOUT`?

Comment (ahopkins) [reply docs/sanic/websocket.rst:55]: Also, it is probably worth adding in a line in the notes here that these setting will have no impact if running in ASGI mode.

Comment (shawnhill) [reply docs/sanic/websocket.rst:58]: Have updated the docs as suggested.

**Comment by shawnhill** [comment]
I'm confused how adding text to the untracked .rst doc files lowered test coverage: https://codecov.io/gh/huge-success/sanic/commit/bcc13178ea96c8fd587abeaa0bedd290db7b0820

**Pull request merged**